doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -633,16 +632,22 @@ must be used.
 
 Multilocale Communication Counts Testing
 ++++++++++++++++++++++++++++++++++++++++
-Writing a communication counts test for multilocale setting is similar to the
-performance counterpart. However, for helper files ``cc-`` label is used instead
-of ``ml-``.
+Communication counts testing is only applicable in a multilocale setting, and it
+is similar to the performance counterpart. However, for helper files ``cc-``
```

Comment:
Sorry, should have caught this earlier: I find the "similar to the performance counterpart" clause confusing.  Maybe "similar to the multilocale performance counterpart" is less so?